### ISSUE TITLE
Forced run - split head, make clear "UI" vs. "daemon"

### ISSUE THREAD

**Issue by licaon-kter** [Catfriend1/syncthing-android#1040]
...forced start is not doing what's expected, the UI messages are confused/ing.

1.26.0.2 F-Droid

The Status UI page says: _"Syncthing is *not running*"_ and lower _"reason: Syncthing *is running* because you forced it to start regardless..."_

No sync happens though.

![188a1c25_C5ujqPtHSyqY6G8Cp3qx7g](https://github.com/Catfriend1/syncthing-android/assets/2282799/f3ce7921-cef8-46f7-8bdf-81cdfe5d0799)

To fix it I need to exit it, then restart it.

**Comment by licaon-kter** [comment]
Attached picture to OP

**Comment by Catfriend1** [comment]
@licaon-kter 

Is the screenshot above the result of the native crashing? This was reported since v1.26.x releases and upstream has merged a PR to fix this, hopefully to be released soon. I agree the messages are weird, but regarding that the native crashing should not be daily matter, I suggest it's okay.

The message "Syncthing is not running" is sent by the native executor which also detects the crash. 
The message in the "reason" section is just to sum up the set run conditions. Confusing yes, but at least we now know why it is ;-).

Maybe we could hide the reasons if a crash was detected.

**Comment by licaon-kter** [comment]
I don't see visible Android crashes, so if it's the daemon under the hood I'd need to check logcat.

**Comment by licaon-kter** [comment]
oh so you have in app logging, will capture one of those when it happens again _(of course I tried now and it didn't crash)_

fyi, when testing, I needed to press restart or kill 2-3 times until it actually was able to start :(

**Comment by Catfriend1** [comment]
@licaon-kter you could also try the latest preview build from the github release page . It contains upstream's fix.

**Comment by EuPhobos** [comment]
After last update this also happens to me too. Syncthing-fork not working anymore for me. Even if device on AC.

**Comment by Catfriend1** [comment]
@EuPhobos try preview?

**Comment by EuPhobos** [comment]
@Catfriend1 yep, trying this file  com.github.catfriend1.syncthingandroid_github_v1.26.1.0_2ee4de2c.apk, same problem.
Before i has f-droid version running fine, today i update app and faced with a this problem.
I export settings, remove f-droid verion, install github version, import settings - same problem.

**Comment by Catfriend1** [comment]
@EuPhobos 

There is a newer release 1.27.0.0

**Comment by EuPhobos** [comment]
@Catfriend1 Oh, my inattention.
1.27 work fine now. Thank you very much! You're best

**Comment by Catfriend1** [comment]
Thanks for reporting back

**Comment by Catfriend1** [comment]
I think this is okay for me, let the head split when the native crashes. If this ever happens, the user is doomed and other parts of the app may also be inaccurate or non-working.

**Comment by Catfriend1** [comment]
For correctness: it gets a verified bug label 😊

**Comment by licaon-kter** [comment]
>  let the head split when the native crashes.

No problem with that, maybe rework the strings a bit to separate "UI" vs "daemon" ?

**Comment by Catfriend1** [comment]
Feel free to submit and translate new strings as required.
